### PR TITLE
Update My Applications to Bootstrap 4

### PR DESCRIPTION
**Pull request by suecarmol**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Migrated the  `My Applications` template  to use Bootstrap 4

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
Bootstrap 3 is EOL

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T293988](https://phabricator.wikimedia.org/T293988)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
Tested manually

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
Screen Shot 2021-11-09 at 19 21 31

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: TWLight/users/templates/users/my_applications.html [hunk 1]
```diff
@@ -1,91 +1,95 @@
...
-        <h4>
-        <a href="{{ app.get_absolute_url }}">
-          {{ app.partner }}
-        </a>
-        </h4>
-        <div class="visible-xs">
-          <span class="label label{{ app.get_bootstrap_class }}">
+  {% include "header_partial_b4.html" %}
+  <div id="main-content">
+    {% include "message_partial.html" %}
```
Comment (jsnshrmn) [inline TWLight/users/templates/users/my_applications.html:8]: nitpick: This partial is outside the main-content div in all other cases. Having it here doesn't break anything on the page, but it does result in the messages having a different position than all the other pages that use the partial.

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
I have one inline nitpick, but this is otherwise ready to merge

**Review by jsnshrmn — APPROVED** [review]

File: TWLight/static/css/new-local.css [hunk 2]
```diff
@@ -405,6 +405,16 @@ COMMON CONTENT BLOCK CSS
     text-decoration: underline;
 }
 
+.twl-secondary-btn{
```
Comment (jsnshrmn) [inline TWLight/static/css/new-local.css:408]: having just gone and cleaned up my own rtl omissions, I just realized that these also need to be added to `new-local.rtl.css`

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
I realized that we overlooked the RTL css.
I've got more cleanup in my outstanding pr, but let's just tackle these new classes in this one.

File: TWLight/static/css/new-local.css (hunk 2 above)
Comment (suecarmol) [reply TWLight/static/css/new-local.css:408]: Good catch! Thanks for spotting that 😄

**Review by jsnshrmn — APPROVED** [review]

**Pull request merged**